### DISCUSSION TITLE
hi,I want to ask some question about how to view the result of the sim

### DISCUSSION THREAD

**Discussion by shengmoxuedaoshi** [cocotb/cocotb#3929] [Q&A]
I have seen a vcd file in my test document ,but it has't upgrade even if I have "make SIM=icarus" for many times ,could you tell me why and how to view the result of cocotb sim,thx.

**Comment by ktbarrett** [comment]
To view a VCD file, I recommend [gtkwave](https://gtkwave.sourceforge.net/). If you are on a Debian-based Linux distro you can just `sudo apt install gtkwave`.
